api/controllers/v1/github.ts

```diff
@@ -109,3 +114,101 @@ export const TriggerBuild = async (event: any = {}, context: any = {}): Promise<
     body: 'Job Queued',
   };
 };
+
+/**
+ * Deletes build artifacts for a given project + branch combination.
+ * @param event
+ */
+export const MarkBuildArtifactsForDeletion = async (event: APIGatewayEvent) => {
```

Comment:
This isn't specific to your PR, but I am wondering how we should handle changes that will exist in both the regular autobuilder and the enhanced autobuilder moving forward. The enhanced autobuilder uses the `api/controllers/v2` directory. Should we maybe create a shared directory as well?

Reply:
Sure, I like the idea for having a shared directory for logic that might not necessarily have differences between v1 and v2. For something like this, maybe we can have a `api/handlers/github.ts` file? Wdyt?